### ISSUE TITLE
Do you need translation help for Chinese?

### ISSUE THREAD

**Issue by favefan** [jeevak001/R6Companion#1]
Maybe I can help you translate something, I'm looking at your "string.xml"

**Comment by jeevak001** [comment]
Yes, I would like help for translations. But I did not think about multiple languages while developing. So I need to pull out data into string.xml for translations to be smooth.

**Comment by favefan** [comment]
ok,I will try to translate some basic phrases in advance from the current "string.xml" file,easy to integrate directly into your translation file after this.thank you for your job.

**Comment by jeevak001** [comment]
Sounds good. I have extracted all data into a file. I will be making some changes to the app and pushing it soon. It will use a new strings.xml file which contains all the data.

**Comment by favefan** [comment]
I started working.but I cloned your project and found some "string.xml" errors, such as "C:\Users\favefan\Desktop\R6Companion-master\app\src\main\res\values\strings.xml: Error: Found item String/k_secondary_weapon more than one time
", maybe you can pay attention

**Comment by favefan** [comment]
ok,i am woring on it

**Comment by littlemadhatter** [comment]
how far are you on working on that i have the idea of how to translate into chinese

**Comment by favefan** [comment]
我已经翻译了一些基础的名词，你可以在"Pull request"里面查看.还有一些探员的介绍可能不是那么快速的翻译.I have translated some basic nouns, you can see in the "Pull request". There are some detectives may not be so fast translation.

**Comment by favefan** [comment]
你如果有好的想法，那当然是极好的。If you have good ideas, of course it is excellent.

**Comment by littlemadhatter** [comment]
ok good

**Comment by littlemadhatter** [comment]
also question

Chinese simplified or Chinese traditional

**Comment by favefan** [comment]
Chinese simplified

发自我的 iPhone

在 2018年5月18日，下午5:33，littlemadhatter <notifications@github.com<mailto:notifications@github.com>> 写道：


also question

Chinese simplified or Chinese traditional
